x-pack/legacy/plugins/siem/public/components/timeline/body/renderers/zeek/zeek_signature.tsx

```diff
@@ -148,7 +148,7 @@ export const TotalVirusLinkSha = React.memo<TotalVirusLinkShaProps>(({ value })
   value != null ? (
     <LinkFlexItem grow={false}>
       <div>
-        <VirusTotalLink link={value}>{value}</VirusTotalLink>
+        <ReputationLink domain={value} overflowIndexStart={1} showDomain={true} />
         <ExternalLinkIcon />
```

Comment:
After changing the `siem:ipReputationLinks` advanced setting to an empty array `[]` to disable external links, the external link icon is still displayed, per the following screenshot:

![empty-icon](https://user-images.githubusercontent.com/4459398/74872069-c30e3f00-5319-11ea-968f-097c2d2e6603.png)

Reply:
Good catch, I'll take this into consideration and covers it with test!